profile/CSIP.xml

```diff
@@ -1223,8 +1222,8 @@
             </requirement>
             <requirement ID="CSIP105" REQLEVEL="SHOULD" EXAMPLES="structMapExample2">
                 <description>
-                    <head>Representation divisions</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">When the transfer consist of representations there are one representation div present for each representation</p>
+                    <head>Representations divisions</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">When the content consist of different representations there are one representation div present for each representation</p>
```

Comment:
When the content is comprised of multiple representations there should be one representation div present for each representation.